vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/facade/ZephyrFacadeFactory.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.zephyr.facade;
+
+import org.vividus.zephyr.model.ZephyrApiType;
+
+public class ZephyrFacadeFactory
+{
+    private final ZephyrFacade zephyrFacade;
+    private final ZephyrScaleFacade zephyrScaleFacade;
+    private final ZephyrApiType zephyrApiType;
+
+    public ZephyrFacadeFactory(ZephyrApiType zephyrApiType,
+                               ZephyrFacade zephyrFacade,
+                               ZephyrScaleFacade zephyrScaleFacade)
+    {
+        this.zephyrApiType = zephyrApiType;
+        this.zephyrFacade = zephyrFacade;
+        this.zephyrScaleFacade = zephyrScaleFacade;
+    }
+
+    public IZephyrFacade getZephyrFacade()
+    {
+        if (zephyrApiType == ZephyrApiType.SQUAD)
+        {
+            return zephyrFacade;
+        }
+        else
+        {
+            return zephyrScaleFacade;
```

Comment:
why not to inject just one proper bean? since only one ZephyrApiType is allowed per execution